app/views/ProjectEdit/Framework/FrameworkDetail/index.tsx

```diff
@@ -332,6 +339,10 @@ function FrameworkDetail(props: Props) {
         showFrameworkCloneModal();
     }, [showFrameworkCloneModal]);
 
+    const handleExportClick = useCallback(() => {
+        showExportModalShown();
+    }, [showExportModalShown]);
+
```

Comment:
I think we can keep showExportModalShown directly instead of creating one more redirect